hbase-server/src/main/java/org/apache/hadoop/hbase/master/MetaFixer.java

```diff
@@ -247,6 +247,10 @@ void fixOverlaps(CatalogJanitor.Report report) throws IOException {
       if (regionInfoWithlargestEndKey != null) {
         if (!isOverlap(regionInfoWithlargestEndKey, pair) ||
             currentMergeSet.size() >= maxMergeCount) {
+          // Log when we cut-off-merge because we hit the configured maximum merge limit.
+          if (currentMergeSet.size() >= maxMergeCount) {
+            LOG.warn("Ran into maximum-at-a-time merges limit={}", maxMergeCount);
```

Comment:
Is `currentMergeSet.size()` also worth in the log message?

Reply:
Not here.